src/Symfony/Component/Validator/Constraints/DivisibleBy.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+/**
+ * @Annotation
+ * @Target({"PROPERTY", "METHOD", "ANNOTATION"})
+ *
+ * @author Colin O'Dell <colinodell@gmail.com>
+ */
+class DivisibleBy extends AbstractComparison
+{
+    const NOT_DIVISIBLE_BY = '6d99d6c3-1464-4ccf-bdc7-14d083cf455c';
+
+    protected static $errorNames = array(
+        self::NOT_DIVISIBLE_BY => 'NOT_MULTIPLE_OF',
+    );
+
+    public $message = 'This value should be divisible by {{ compared_value }}.';
```

Comment:
Would be nice to also add translations for this string.

Reply:
The message is also not all that clear from a user's perspective. To a developer "8 is divisible by 2" makes sense, but a mathematician would say "so is 7.6543", and a common user would just be like "erm keep the math out of my way".

I'd stick with the more generic `This value must be a multiple of {{ compared_value }}.`

Reply:
I have updated the message as requested and force-pushed the changes.

Reply:
I have also added translation strings in English and Spanish.